pkg/workloads/console/integration/integration_test.go

```diff
@@ -236,7 +288,15 @@ var _ = Describe("Console", func() {
 			Expect(err).NotTo(HaveOccurred(), "failed to update console")
 
 			waitForSuccessfulReconcile(1)
-			// TODO: check that the 'already exists' event was logged
+
```

Comment:
Actually, how do you feel about PR'ing these commented pieces and leaving the PR open, with an explanation of what the goal is? Then we can unclutter the tests as is while tracking the work there.

Reply:
That's actually a great idea; doing that now!